api/sql/schema/750$rule.decision.fetch.sql

```diff
@@ -123,6 +130,22 @@ BEGIN
         c.priority,
         l.limitId
 
+    IF @isTransactionValidate = 0 AND @isMaskChecked = 1 AND @sourceCheckAmount <> 0 AND @sourceCheckAmount <= @amount
+        BEGIN          
+           IF ((@checkMask & @sourceCheckMask) = @sourceCheckMask)              
+               SET @checkSuccess = 1
+            ELSE 
+               SET @checkSuccess = 0    
+        END
+    ELSE 
+        SET @checkSuccess = 1
+    
+    IF @checkSuccess = 0
```

Comment:
This is not needed. Instead in the next checks, failed credentials mask, should affect the error code that is returned.